src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -577,7 +577,7 @@
     <value>When on multiple lines</value>
   </data>
   <data name="Show_items_from_unimported_namespaces" xml:space="preserve">
-    <value>Show items from unimported namespaces (experimental)</value>
+    <value>Show items from unimported namespaces</value>
```

Comment:
By removing the 'experiment' flag, does that mean we are plan to use the cancellation data to see if we can enable this by default for all the user?

Reply:
This feature has been stabilized for a few releases, so functionally we think we are ready to move it out of the experimental state. However, current implementation still incurs too much of perf hit for many users to justify enabling it by default. Various things contributed to this, e.g. OOP cost, some existing editor APIs is not designed for handling large amount of items, etc. We will keep making improvement and monitoring the telemetry, hopefully we can turn it on by default in a future release.